src/datasets/domain/models/CitationFormats.ts

```diff
@@ -0,0 +1,7 @@
+export enum CitationFormats {
```

Comment:
```suggestion
export enum CitationFormat {
```
To be consistent with other enums, I think it's better to use a singular name, rather than plural